src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -19,15 +19,23 @@ export default function ColorPickerControl({
 }: Props): JSX.Element | null {
     const paletteRef = React.useRef<HTMLDivElement>(null);
     const [isColorPickerToggled, setIsColorPickerToggled] = React.useState(false);
-    const [isPaletteActive, handlers] = useAttention();
+    const { current: paletteEl } = paletteRef;
+
+    const handlePaletteBlur = ({ relatedTarget }: React.FocusEvent<HTMLButtonElement>): void => {
+        const targetElement = relatedTarget as HTMLButtonElement;
+        if (targetElement.parentElement && targetElement.parentElement.parentElement !== paletteEl) {
```

Comment:
The `parentElement` calls here break encapsulation by making some assumptions about the DOM that may eventually change and break unexpectedly. Can we use `!paletteEl.contains(targetElement)`, instead?